inspirehep/dojson/experiments/fields/bd1xx.py

```diff
@@ -171,6 +171,10 @@ def date_started(self, key, value):
     values = force_force_list(value)
 
     for val in values:
+        if val.get('q'):
+            self['date_proposed'] = val.get('q')
+        if val.get('r'):
+            self['date_approved'] = val.get('r')
```

Comment:
These can just be `self['date_proposed'] = val.get('q')`, since `None` values are cleaned later.